localMatchApp/test/logintest.cpp

```diff
@@ -0,0 +1,83 @@
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+#include <iostream>
+#include <string>
+#include "Login.h"
+using namespace testing;
+
+TEST ( LoginTest, TestPasswordGoodCases ) {
```

Comment:
provide a test description of what you are trying to verify

Reply:
Done.